python/ray/_raylet.pyx

```diff
@@ -748,7 +748,7 @@ cdef execute_task(
                                         core_worker.get_current_task_id()),
                                     exc_info=True)
                     else:
-                        logger.info("Task failed with unretryable exception:"
+                        logger.debug("Task failed with unretryable exception:"
```

Comment:
@rkooo567 Do you remember why we made the above log statement (`"Task failed with retryable exception"`) info-level originally? That seems like it would create a good bit of noise for what should be a transparent retry.

Reply:
Hmm cc @jjyao who seemed to add this. I guess there's nothing to lose by making it debug (since we raise an exception anyway), but just in case

Reply:
@rkooo567 In this comment, I'm talking about the log statement above this one, the `"Task failed with retryable exception"`, when we retry tasks. I couldn't comment on it directly since it's not changed in this PR.

Reply:
Imo, it should be okay to make that debug as well. But I will wait for @jjyao just in case.